src/resources/postcss/views/_month.pcss

```diff
@@ -4,4 +4,38 @@
  * @package TribeEventsCalendar
  *
  */
-.tribe-events {}
\ No newline at end of file
+.tribe-events {
+
+	.tribe-events-calendar-month__event-multiday {
+		background-color: rgba(var(--color-accent-primary), .1);
+		border-radius: 6px;
```

Comment:
Let's apply the border radius to the outer wrapper and all the border styles to the inner wrapper. This prevents the blue border from curving, as we're seeing in the screenshot.